packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -220,6 +232,59 @@ const Breakouts = WebexPlugin.extend({
 
     this.breakouts.set(Object.values(breakouts));
   },
+
+  /**
+   * Make enable breakout resource
+   * @returns {Promise}
+   */
+  touchBreakout() {
+    // @ts-ignore
+    return this.webex
+      .request({
+        method: HTTP_VERBS.POST,
+        uri: this.breakoutServiceUrl,
+        body: {
+          locusUrl: this.locusUrl,
+        },
+      })
+      .catch((err) => {
+        LoggerProxy.logger.error(
+          `Meeting:request#touchBreakout --> Error provisioning error ${err}`
+        );
+        throw err;
+      });
+  },
+
+  /**
+   * Make the meeting enbale or disable breakout session
+   * @param {boolean} enable
+   * @returns {Promise}
+   */
+  async toggleBreakout(enable) {
+    if (this.enableBreakoutSession === undefined) {
+      await this.touchBreakout().then(() => {
```

Comment:
question: where is `this.enableBreakoutSession` set to true? Is it that as a result of sending the requests to Locus we get a Locus update and updateBreakout() is called and enableBreakoutSession is set then?

Reply:
Yes, Locus will update and updateBreakout() set the enableBreakoutSession